syntax = "proto3";

package users;

// The user service
service UserService {
  rpc GetUser (UserRequest) returns (UserReply);
  rpc CreateUser (CreateUserRequest) returns (UserReply);
}

message UserRequest {
  int32 id = 1;
}

message CreateUserRequest {
  string name = 1;
  string email = 2;
}

message UserReply {
  int32 id = 1;
  string name = 2;
  string email = 3;
}
